src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.containers') {
+    .clr-form-control {
+        margin-top: $clr-forms-baseline * 4;
+        display: flex;
+        flex-direction: column;
+
+        .clr-control-label {
+            display: block;
+            color: $clr-forms-label-color;
+            font-size: calc(#{$clr-forms-baseline} * 2 + 1px);
+            font-weight: 600;
+            line-height: $clr-forms-baseline * 3;
+        }
+
+        .clr-control-container {
+            display: inline-block;
+
+            .clr-subtext {
+                display: block;
+                font-size: calc(#{$clr-forms-baseline} * 2 - 1px);
+                line-height: $clr-forms-baseline * 2;
+                color: $clr-forms-subtext-color;
+                margin-top: $clr-forms-baseline;
+            }
+
+            .clr-validate-icon {
+                width: $clr-forms-icon-dimension;
+                height: $clr-forms-icon-dimension;
+                margin-left: -$clr-forms-icon-dimension;
+                display: none;
+            }
+
+            &.clr-error {
+                .clr-validate-icon {
+                    color: $clr-forms-input-invalid-color;
+                    display: inline-block;
+                }
+
+                .clr-subtext {
+                    color: $clr-forms-input-invalid-color;
+                }
+            }
+        }
+
+        &.clr-form-control-disabled {
+            .clr-control-label {
+                color: $clr-input-disabled-color;
+            }
+        }
+    }
+
+    .clr-form-horizontal {
+        .clr-control-label {
+            width: $clr-forms-baseline * 32;
+        }
+        &.row {
+            .clr-control-label {
+                width: auto;
+            }
+        }
+
+        .clr-form-control {
+            display: flex;
+            flex-direction: row;
+        }
+    }
+
+    .clr-form-compact {
+        .clr-form-control > .clr-control-label {
+            width: $clr-forms-baseline * 32;
+            min-width: $clr-forms-baseline * 32;
+        }
+        .clr-form-control.row > .clr-control-label {
+            width: auto;
+        }
+
+        .clr-control-label {
+            line-height: $clr-forms-baseline * 4;
+        }
+
+        .clr-form-control {
+            display: flex;
+            flex-direction: row;
+            padding-top: $clr-forms-baseline * 2;
+
+            .clr-control-container {
+                display: flex;
+            }
+            
+            .clr-subtext {
```

Comment:
This is the one I'm mentioning in my comment above. Wouldn't `clr-form-compact .clr-subtext` be just as good for us and way easier to override for our consumers?